cli/tools/lint.rs

```diff
@@ -373,3 +415,57 @@ fn sort_diagnostics(diagnostics: &mut Vec<LintDiagnostic>) {
     }
   });
 }
+
+fn get_rules_from_config(
+  maybe_lint_config: Option<&LintConfig>,
+) -> Result<Vec<Box<dyn LintRule>>, AnyError> {
+  let configured_rules = if let Some(lint_config) = maybe_lint_config {
+    let mut filtered_rules = HashMap::new();
+
+    if let Some(tags) = &lint_config.rules.tags {
```

Comment:
I don't have an opinion on this, but if someone doesn't specify "tags", should we default to the default of "recommended"? So if they don't want the default and want nothing they would have to specify `"tags": []`

I'm thinking a common path might be someone doing:

```json
{
  "lint": {
    "exclude": ["some-rule"]
  }
}
```

...and in doing so, they would have to remember to add `"tags": ["recommended"]`. Alternatively, perhaps we can suggest this to the user if they do the above (see my comment below).

Reply:
Ok thanks! I misread the code then.